src/Grid/grid.jl

```diff
@@ -109,33 +109,33 @@ getdim(::Cell{dim}) where dim = dim
 abstract type AbstractTopology end
 
 struct ExclusiveTopology <: AbstractTopology
-    vertex_to_cell::Vector{Vector{Int}}
+    vertex_to_cell::Dict{Int,Vector{Int}}
     cell_neighbor::Vector{EntityNeighborhood{CellIndex}}
     face_neighbor::SparseMatrixCSC{EntityNeighborhood,Int}
     vertex_neighbor::SparseMatrixCSC{EntityNeighborhood,Int}
     edge_neighbor::SparseMatrixCSC{EntityNeighborhood,Int}
-    vertex_vertex_neighbor::Vector{EntityNeighborhood{VertexIndex}}
+    vertex_vertex_neighbor::Dict{Int,EntityNeighborhood{VertexIndex}}
```

Comment:
What was the rationale behind this change?

Reply:
Vertices do not have a linear index for higher order cases. Up till now I used the vector index and mapped it to the neighbors, but this isn’t possible for e.g. a quadratic quadrilateral where your First vertex is global id 1 and your second vertex is global id 3. that’s why we need an actual map